### PR TITLE
Fortify scan enabled

### PR DESCRIPTION
**Pull request by divisathyan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-9381

### Change description ###
The changes are in this PR but the same are implemented on nightly branch

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -40,6 +40,8 @@ def secrets = [
     secret('robotics-email-scottish-to', 'ROBOTICS_EMAIL_SCOTTISH_TO'),
     secret('robotics-email-pip-ae-to', 'ROBOTICS_EMAIL_PIP_AE_TO'),
     secret('AppInsightsInstrumentationKey', 'APPINSIGHTS_INSTRUMENTATIONKEY'),
+    secret('fortify-on-demand-username','FORTIFY_USER_NAME'),
+    secret('fortify-on-demand-password','FORTIFY_PASSWORD')
```
Comment (MSancaktutar) [inline Jenkinsfile_nightly:44]: These lines should be removed. The secrets should be set on sscs-aat key vault.

File: charts/sscs-evidence-share/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ description: SSCS Evidence Share
 home: https://github.com/hmcts/sscs-evidence-share
 apiVersion: v2
 appVersion: "1.0"
-version: 0.0.39
+version: 0.0.40
```
Comment (MSancaktutar) [inline charts/sscs-evidence-share/Chart.yaml:6]: no need really.

File: charts/sscs-evidence-share/values.yaml [hunk 3]
```diff
@@ -27,6 +27,8 @@ java:
         - AppInsightsInstrumentationKey
         - ld-sdk-key
         - sendgrid-api-key
+        - fortify-on-demand-username
+        - fortify-on-demand-password
```
Comment (MSancaktutar) [inline charts/sscs-evidence-share/values.yaml:31]: no need for these.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
please change branch name to have a prefix of 'nightly', like `nightly-SSCS-9381-fortify`, and remove the undesirable lines for secrets/env vars.

File: charts/sscs-evidence-share/Chart.yaml (hunk 2 above)
Comment (divisathyan) [reply charts/sscs-evidence-share/Chart.yaml:6]: This was automatically bumped, reverting it

File: charts/sscs-evidence-share/values.yaml (hunk 3 above)
Comment (divisathyan) [reply charts/sscs-evidence-share/values.yaml:31]: Removed

File: Jenkinsfile_nightly (hunk 1 above)
Comment (divisathyan) [reply Jenkinsfile_nightly:44]: Removed

**Comment by divisathyan** [comment]
Closing the PR as #1048 is created as a duplicate

**Pull request closed without merging**